python_configure.bzl

```diff
@@ -314,6 +324,51 @@ def _python_autoconf_impl(repository_ctx):
     """Implementation of the python_autoconf repository rule."""
     _create_local_python_repository(repository_ctx)
 
+def _find_python_config(repository_ctx, python_bin):
```

Comment:
Minor nitpick: Can you move these additions above the definition of `_python_autoconf_impl()` (i.e. line 323)?

It's nice to keep the rule and it's implementation next to each other, as it makes it slightly easier for a newcomer to start reading.